engine/utils.go

```diff
@@ -0,0 +1,146 @@
+package engine
+
+import (
+	"fmt"
+	specv1 "github.com/baetyl/baetyl-go/spec/v1"
+)
+
+const (
+	appDataHostPath = "/var/lib/baetyl/app-data"
```

Comment:
这里加个TODO：when baetyl-init applies the beatyl-core deployment yaml， pass the app data
path in host through env to baetyl-init, does not hardcode the value here because the host path may be changed during baetyl installation